react/features/connection-indicator/components/web/ConnectionIndicator.js

```diff
@@ -225,7 +230,8 @@ class ConnectionIndicator extends AbstractConnectionIndicator<Props, AbstractSta
      */
     _renderIcon() {
         if (this.props._connectionStatus
-            === JitsiParticipantConnectionStatus.INACTIVE) {
+            === JitsiParticipantConnectionStatus.INACTIVE
+            && !this.props._connectionIndicatorInactiveDisabled) {
```

Comment:
Is this the right place to do this? IIRC there is another place which checks if an icon has to be rendered at all. Can you check?